packages/garbo/src/tasks/barfTurn.ts

```diff
@@ -955,6 +957,38 @@ const BarfTurnTasks: GarboTask[] = [
     combat: new GarboStrategy(() => Macro.meatKill()),
     spendsTurn: () => globalOptions.target.attributes.includes("FREE"),
   },
+  {
+    name: "Liana Parachute",
+    ready: () =>
+      (sober() ||
+        (have($item`Drunkula's wineglass`) &&
+          canEquip($item`Drunkula's wineglass`))) &&
+      CrepeParachute.have() &&
+      shouldCheckParachute() &&
+      questStep("questL11Worship") > 3 &&
+      have($item`antique machete`), // TODO Support other machete's
+    completed: () => have($effect`Everything looks Beige`),
+    outfit: () => freeFightOutfit({ weapon: $item`antique machete` }),
+    do: () => CrepeParachute.fight($monster`dense liana`),
+    combat: new GarboStrategy(() =>
+      Macro.abortWithMsg(
+        "Did not instantly kill the Liana, check what went wrong",
+      ),
+    ),
+    prepare: () => {
+      if (!sober()) {
+        freeFightOutfit({ offhand: $item`Drunkula's wineglass` }).dress();
+      }
+      withChoice(785, 6, () =>
+        adv1($location`An Overgrown Shrine (Northeast)`, -1, ""),
```

Comment:
long-term, unfortunately, I think we'll have to split this prepare into a separate task to handle holiday wanderers properly. But again, short-term, I'm just going to merge this I think